icomoon/style.css

```diff
@@ -0,0 +1,63 @@
+@font-face {
+  font-family: "icomoon";
+  src: url("fonts/icomoon.woff?adhf8k") format("woff");
+  font-weight: normal;
+  font-style: normal;
+  font-display: block;
+}
+
+[class^="icon-"],
+[class*=" icon-"] {
+  /* use !important to prevent issues with browser extensions that change fonts */
+  font-family: "icomoon" !important;
+  speak: never;
+  font-style: normal;
+  font-weight: normal;
+  font-variant: normal;
+  text-transform: none;
+  line-height: 1;
+
+  /* Better Font Rendering =========== */
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+.icon-ChevronDown:before {
+  content: "\e900";
+}
+.icon-f-icon_footer:before {
+  content: "\e901";
+}
+.icon-icon_1:before {
```

Comment:
Will you be able to recall what does this icon-icon mean after several months?

Reply:
I've changed all icon-icon elements, and set new name. For example: .icon-cloud-1